R/zzz.R

```diff
@@ -1,3 +1,3 @@
 ##' @useDynLib dust2, .registration = TRUE
-cache <- list2env(list(models = list()),
+cache <- list2env(list(generators = list()),
```

Comment:
A cache used to have models, but now has generators, rather than systems...

Reply:
this really should have been generators previously too